### PR TITLE
Use mapit for geolocation and shapes in downloads

### PR DESCRIPTION
**Pull request by longhotsummer**

### CODE REVIEW

**Comment by longhotsummer** [comment]
@xybrnet

File: wazimap_za/geo.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        """
+        Returns a list of geographies containing this point.
+        """
+        resp = requests.get(self.MAPIT_API_URL + '/point/4326/%s,%s' % (longitude, latitude))
+        resp.raise_for_status()
+
+        geos = []
+        for feature in resp.json().itervalues():
+            try:
+                geo = self.get_geography(feature['codes']['MDB'],
```
Comment (xybrnet) [inline wazimap_za/geo.py]: Where is get_geography?

Comment (xybrnet) [inline wazimap_za/geo.py]: Ah, got it.

**Comment by xybrnet** [comment]
:+1:

**Pull request merged**